app/src/androidTest/java/com/swent/assos/AssoDetailsTest.kt

```diff
@@ -53,4 +77,37 @@ class AssoDetailsTest : SuperTest() {
       }
     }
   }
+
+  @Test
+  fun joinAssociation() {
+    DataCache.currentUser.value.associations = emptyList()
+    run {
+      ComposeScreen.onComposeScreen<AssoDetailsScreen>(composeTestRule) {
+        step("Join association") {
+          joinButton {
+            assertIsDisplayed()
+            performClick()
+          }
+        }
+      }
+    }
+  }
+
+  @Test
+  fun testThatTheEventIsDisplayed() {
+    run {
+      ComposeScreen.onComposeScreen<AssoDetailsScreen>(composeTestRule) {
+        step("Check that the event is displayed") { eventItem { assertIsDisplayed() } }
+      }
+    }
+  }
+
+  /*@Test
+  fun testThatTheNewsIsDisplayed() {
```

Comment:
Try not to leave the test commented, there is no reason it shouldn't work, maybe try to review the AssoDetailsScreen.